nodejs-server/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "showcase-nodejs-echo-server",
+  "version": "0.0.1",
+  "description": "",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src"
+  ],
+  "license": "Apache-2.0",
+  "keywords": [],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "check": "gts check",
```

Comment:
it's a total nit, but we've been using `lint` here since it's a little more community standard

Reply:
Changed to `lint`. I actually used `npx gts init` so unless this will be changed in gts 1.0, it's what `gts init` writes.